eliot/prettyprint.py

```diff
@@ -0,0 +1,73 @@
+"""
+API and command-line support for human-readable Eliot messages.
+"""
+
+from __future__ import unicode_literals
+
+from datetime import datetime
+from sys import stdin, stdout, argv
+from json import loads
+
+from six import text_type as unicode, PY2
+
+
+def pretty_print(message):
```

Comment:
Pretty minor, but `pprint.pprint` actually prints the result to stdout.  This is more like `pprint.pformat` - iow, it doesn't pretty print, it pretty formats.